frontend/document/pdfdocument.lua

```diff
@@ -59,7 +59,7 @@ function PdfDocument:layoutDocument(font_size)
 end
 
 local default_font_size = 22
--- the koptreader config goes from 0.1 to 3.0, but we want a regular font size
+-- the koptreader config goes from 0.2 to 2.0, but we want a regular font size
```

Comment:
Please don't change this comment. I didn't write the values correctly by mistake. ;-)

Reply:
This is a code comment, not a UI comment.

Reply:
Sorry, don't understand.
https://github.com/koreader/koreader/blob/827581ac728330b38853450ff59c3afca00fe488/frontend/ui/data/koptoptions.lua#L10

Reply:
Left 3, right 2
<img src=https://user-images.githubusercontent.com/202757/229475218-8d7f46ec-96b4-4105-a4f4-9df68a572a72.png width=40%> <img src=https://user-images.githubusercontent.com/202757/229475226-98fe1baf-77ed-4fbc-a581-974035352e93.png width=40%>

Reply:
PS Those larger and smaller sizes are accessible through the UI too through larger/smaller. But they're too large and small respectively to eat up valuable UI space.